.gitignore

```diff
@@ -100,5 +100,6 @@ contrib/modellauncher/modellauncher
 # don't checkin renv
 /renv/
 
-# ignore IP mapping to lat/lon (is about 65MB)
-shiny/dbsync/IP2LOCATION-LITE-DB5.BIN
+# folder with the data folders for the docker stack
+/volumes
```

Comment:
Would it make sense to move this to be `docker/volumes` to make its purpose clearer and avoid cluttering the root directory? (I know gitignore is the least of the places it'll need changing, just writing it where I thought of it...)

Reply:
if so we should move all the volumes in docker-compose.dev.yml to be using "${PWD}/docker/volumes/XYZ"